### DISCUSSION TITLE
[Q&A] More than one server for GLPI-agent 1.4

### DISCUSSION THREAD

**Discussion by ssoubie86** [glpi-project/glpi-agent#342] [Q&A]
### Documentation & bug reporting acknowledgment

Yes, I read it

### Describe your problem

Hi everybody,

I didn't find any solution about this problem. 
With FusionInventory, it's possible to specify 2 GLPI server for inventory. But is it possible with GLPI-Agent.
I depoyed with success GLPI-Agent 1.4 via GPO on one of my servers, but for the second one, there is a problem.

Have you some information ?
Thanks

**Comment by g-bougard** [comment]
Hi @ssoubie86 
as GLPI-Agent is a fork of FI Agent, yes, it supports specifying 2 GLPI servers like FI supports.
You have to set them as a comma separated list for the `SERVER` parameter.
You're telling there's a problem with the second one but not sharing any info of the problem itself.
While using GPO, sometime people use a PowerShell script to deploy and the installer is known to fail in that context.
If you're still not, you should probably use the provided VBS as explained in doc:
https://glpi-agent.readthedocs.io/en/latest/installation/index.html#large-installations
And don't forget to read the online doc about configuring the agent when installing on windows:
https://glpi-agent.readthedocs.io/en/latest/installation/windows-command-line.html
